src/test/java/de/tum/in/www1/artemis/ExerciseIntegrationTest.java

```diff
@@ -550,7 +550,7 @@ public void testResetExercise() throws Exception {
         List<Course> courses = database.createCoursesWithExercisesAndLectures(true);
         for (Course course : courses) {
             for (Exercise exercise : course.getExercises()) {
-                request.delete("/api/exercises/" + exercise.getId() + "/reset", HttpStatus.OK);
+                request.postWithoutResponseBody("/api/exercises/" + exercise.getId() + "/delete", HttpStatus.OK, null);
```

Comment:
Why is it necessary to change this call from reset to delete here?

Reply:
This was a stupid mistake i already fixed ^^'